packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -163,11 +167,64 @@ async function handleGrant(config: ReleaseGoldConfig, currGrant: number) {
       from: fromAddress,
     },
   ])
+  const [initializerAbiRG, transferImplOwnershipAbiRG] = checkRGImplementationAbi(
+    ReleaseGold,
+    'ReleaseGold'
+  )
   console.info('  Deploying ReleaseGoldProxy...')
   const releaseGoldProxy = await retryTx(ReleaseGoldProxy.new, [{ from: fromAddress }])
   console.info('  Deploying ReleaseGold...')
   const releaseGoldInstance = await retryTx(ReleaseGold.new, [{ from: fromAddress }])
 
+  // We need to fund the RG implementation instance in order to initialize it.
```

Comment:
What do you think about extracting this into a helper function so this one becomes more readable?

Reply:
Good idea